eng/Versions.props

```diff
@@ -25,11 +25,11 @@
     <VersionFeature21>30</VersionFeature21>
     <VersionFeature31>32</VersionFeature31>
     <VersionFeature50>17</VersionFeature50>
-    <VersionFeature60>29</VersionFeature60>
-    <VersionFeature70>18</VersionFeature70>
+    <VersionFeature60>$([MSBuild]::Add($(VersionFeature), 29))</VersionFeature60>
```

Comment:
We're at 8.0.300. The current unreleased implicit version is 30 so this is off by one. See the 2xx branch how it adds 25 and the version is 205 currently: https://github.com/dotnet/installer/blob/release/8.0.2xx/eng/Versions.props#L29C1-L30C82

Reply:
Thanks, I made a fix here: https://github.com/dotnet/installer/pull/19532.